draft-ietf-quic-transport.md

```diff
@@ -860,39 +860,95 @@ different IP or port at either endpoint, due to NAT rebinding or mobility, as
 described in {{migration}}.  Finally a connection may be terminated by either
 endpoint, as described in {{termination}}.
 
+
+## Matching Packets to Connections
+
+Incoming packets are classified on receipt.  Packets can either be associated
+with an existing connection, be discarded, or - for servers - potentially create
+a new connection.
+
+A packet that can be identified as belonging to an existing connection is
+handled according to the current state of that connection.  Packets are
+associated with existing connections using connection ID if it is present; this
+might include connection IDs that were advertised using NEW_CONNECTION_ID
+({{frame-new-connection-id}}).  Packets without connection IDs and long-form
+packets for connections that have incomplete cryptographic handshakes are
+associated using the tuple of source and destination IP addresses and ports.
+
+A packet that uses the short header form could be associated with an existing
```

Comment:
This isn't accurate. A server could use the long header for 1-RTT encypted packets, and 0-RTT encrypted packets are long header.

Reply:
Actually, it is correct, but it wasn't clear what it was referring to.  It refers to the long-form packets and short-form headers sent by the server as it completes the handshake.  It sends a long header for the handshake, then short headers for packets that follow that.  The packets with a short header can arrive before the packet containing the handshake.

I will expand on this a little.